src/three-domain-secure/component.jsx

```diff
@@ -102,7 +102,7 @@ export function getThreeDomainSecureComponent() : TDSComponent {
                                 return onError(err);
                             }
 
-                            return value(true);
+                            return value(err, result);
```

Comment:
@bluepnume we found that this `onSuccess()` prop is breaking with zoid@9.0.81. It seems like the decorator is getting called twice now and the second time it breaks. Calling `value()` with the correct args seems to fix it.

What are your thoughts on this one? Fix in zoid or common-components?

cc: @westeezy @wsbrunson